### PR TITLE
Feat(Identity): Unify Alias and Identity view

### PR DESCRIPTION
**Pull request by eike-hass**
# Description of change

Unify the identity resolver and Alias Address view. Add domain name verification for linked domains.

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Enhancement (a non-breaking change which adds functionality)

## How the change has been tested

Ran through 

## Change checklist

Add an `x` to the boxes that are relevant to your changes, and delete any items that are not.

- [X] My code follows the contribution guidelines for this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by msarcev** [comment]
image
Tried to load a random Alias address and it exploded here. Not sure about the case, but seams like we need to be more careful using `hexToUtf8`.

**Review by msarcev — APPROVED** [review]

**Comment by msarcev** [comment]
The "disabled" state of the tab is not working correctly:
If i go to this address
http://localhost:3000/testnet/addr/rms1prkreqc08fsq0dxg6qlnu5u3sjx4z4x0g830kuej8q63s2nnzypsvynznx0?tab=State
or actually through search with this did:iota:rms:0xec3c830f3a6007b4c8d03f3e5391848d5154cf41e2fb73323835182a73110306
it find the address and goes to DID tab, but the tab is “disabled” (grey) for some reason. And if you go to some other tab, you cannot go back!

File: client/src/app/routes/stardust/AddressState.ts [hunk 1]
```diff
@@ -51,6 +54,9 @@ export interface IAddressState {
     tokensCount: number;
     nftCount: number;
     associatedOutputCount: number;
+    aliasContainsDID: boolean;
```
Comment (msarcev) [inline client/src/app/routes/stardust/AddressState.ts:57]: Please add "defaults" for the new fields in `initialState`. I suggest making the type of resolvedDID `IIdentityStardustResolveResponse | null` and make it `null` by default for easier checks if it's populated.

Comment (eike-hass) [reply client/src/app/routes/stardust/AddressState.ts:57]: added the default values 💪

**Comment by eike-hass** [comment]
> The "disabled" state of the tab is not working correctly: If i go to this address http://localhost:3000/testnet/addr/rms1prkreqc08fsq0dxg6qlnu5u3sjx4z4x0g830kuej8q63s2nnzypsvynznx0?tab=State or actually through search with this did:iota:rms:0xec3c830f3a6007b4c8d03f3e5391848d5154cf41e2fb73323835182a73110306 it find the address and goes to DID tab, but the tab is “disabled” (grey) for some reason. And if you go to some other tab, you cannot go back!

fixed in the latest commit, not sure how that could have ever worked previously 🤔

**Review by msarcev — APPROVED** [review]

**Pull request merged**